day00/ex09/ex09.md

```diff
@@ -0,0 +1,62 @@
+# Exercise 09 - Mean Squared Error as linear cost function
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex09              |
+|   Files to turn in :    |  linear_mse.py     |
+|   Forbidden function :  |                    |
+|   Remarks :             |  n/a               |
+
+**Objective:**
+You must implement the following formula as a function:  
+
+$$
+MSE = \frac{1}{m}\sum_{i=1}^{m}(h_{\theta}(x_i) - y_i)^2
+$$
+
+where:
+- $x$ is a matrix of dimension m * n,
+- $x_i$ is a vector of dimension n,
+- $\theta$ is a vector of dimension n,
+- $(h_{\theta}(x_i)$ is the dot product of the vectors $\theta$ and $x_i$,
```

Comment:
... of the vectors $\theta^{T}$ and ...

Reply:
Not in this case : the dot product imply doing a transpose : a dot b = a^{T}b

No ?

Reply:
if it implicitly does the tranpose, then yes